sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/domain/ResourceTreeNode.java

```diff
@@ -56,7 +56,7 @@ public static ResourceTreeNode fromNodeVoList(List<NodeVo> nodeVos) {
             ResourceTreeNode node = fromNodeVo(vo);
             map.put(node.id, node);
             // real root
-            if (node.parentId == null) {
+            if (node.parentId == null || node.parentId.isEmpty()) {
```

Comment:
Maybe this can be simplified as `StringUtil.isEmpty(node.parentId)`.

Reply:
yeah, but I think keep domain clean will be better